### PR TITLE
Gmpe config

### PR DESCRIPTION
**Pull request by shot0829**
added entry for openquake config. an example/default config is also added

### CODE REVIEW

File: empirical/scripts/calculate_empirical.py [hunk 1]
```diff
@@ -162,6 +163,12 @@ def calculate_empirical(
     station_names = [site.name for site in sites] if stations is None else stations
 
     for im in ims:
+        # read openquake parameter config
+        # if oq_model_param_config == None, default config file will be used
+        oq_model_params_dict = empirical_factory.get_oq_model_params(
+            oq_model_param_config
+        )
```
Comment (jpa198) [inline empirical/scripts/calculate_empirical.py:170]: This doesn't seem to change per im, so can move outside the for loop?

**Review by jpa198 — CHANGES_REQUESTED** [review]
Heh, that was a small enough change in the end
Only one thing though

**Review by jasonmotha — COMMENTED** [review]
Our locally implemented models will also need to be able to support these custom configurations too. Shouldn't be too much to extend this.

Perhaps as well documenting on https://wiki.canterbury.ac.nz/display/QuakeCore/Empirical+Engine (either in an existing or current table) what parameters have options available.

**Review by jpa198 — APPROVED** [review]
Looks good

**Review by jasonmotha — APPROVED** [review]
Looks good. @shot0829 Are you able to do the documentation as to which models can accept which parameters and which values are valid?

**Pull request merged**